lib/portlayer/storage/storage.go

```diff
@@ -150,6 +159,9 @@ type DataSource interface {
 	//     nfs volume:  		 XDR-client
 	//     via guesttools:  	 toolbox client
 	Source() interface{}
+
+	// Given a filter spec
+	Stat(op trace.Operation, spec *archive.FilterSpec) (*FileStat, error)
```

Comment:
changed to this : `// Stat stats the filesystem target indicated by the last entry in the given Filterspecs inclusion map`